pkg/gameservers/controller.go

```diff
@@ -82,7 +82,8 @@ func NewController(
 	kubeInformerFactory informers.SharedInformerFactory,
 	extClient extclientset.Interface,
 	agonesClient versioned.Interface,
-	agonesInformerFactory externalversions.SharedInformerFactory) *Controller {
+	agonesInformerFactory externalversions.SharedInformerFactory,
+) *Controller {
```

Comment:
For some reason, I find this being on the next line kinda weird. I'm assuming you did this on purpose :wink: